(ESPSTUDIO-8275)

mtlljm opened the pull request:
Plugin deployment option for different namespaces on Viya 
make the readme more VIYA focused

mtlljm committed:
(ESPSTUDIO-8275)
Plugin deployment option for different namespaces on Viya
make the readme more VIYA focused

Change-Id: I514c15ed4eececcde3d02899cb315fac5cbef826

mtlljm committed:
(ESPSTUDIO-8275)
Plugin deployment option for different namespaces on Viya
make the readme more VIYA focused

Change-Id: I68ff247d0a94cfe1e060663fd2a152c58e54376b

katjam-sas commented on README.md:
```diff
@@ -73,18 +73,18 @@ This section is relevant only to internal users at SAS.
-* A Grafana deployment with the name `grafana`, running in the same namespace as SAS Event Stream Processing.
-* It is recommended to have an Ingress for the Grafana deployment.
+* SAS Event Stream Processing running in the SAS Viya platform.
+* A Grafana deployment with the name `grafana`, this can be on a separate namespace to SAS Event Stream Processing.
```
How about this:
A Grafana deployment with the name `grafana`. The Grafana deployment can be in the same namespace as SAS Event Stream Processing or in a separate namespace.

katjam-sas commented on README.md:
```diff
@@ -73,18 +73,18 @@ This section is relevant only to internal users at SAS.
  * Modify the Grafana deployment by adding the GF_INSTALL_PLUGINS environment variable to enable Grafana to install the plug-in.
- * Configure a new `grafana.ini` file to enable OAuth authentication.
- * Configure Grafana as an OAuth client with the chosen OAuth provider. Users of Grafana are directed to use the OAuth login page.
+ * Create a new `grafana.ini` file to enable OAuth authentication, this will overwrite any existing Grafana configuration.
```
How about this:
Create a new `grafana.ini` file to enable OAuth authentication. Creating this file overwrites any existing Grafana configuration.

katjam-sas commented on README.md:
```diff
@@ -101,29 +101,22 @@ Installation scripts are provided to install the plug-in and configure Grafana.
 
 4. Run `configure-grafana.sh`, adjusting the command to specify the following variables:
-   - The Kubernetes namespace in which SAS Event Stream Processing is installed _namespace_.
+   - The Kubernetes namespace in which SAS Event Stream Processing is installed _esp-namespace_.
```
Add a comma after "installed", for consistency with the next bullet and with step 5.

katjam-sas commented on README.md:
```diff
@@ -101,29 +101,22 @@ Installation scripts are provided to install the plug-in and configure Grafana.
-   - The _oauth-provider_ of the environment. Select one of the following options: `uaa`, `keycloak` or `viya`.
-   - (Optional) The Kubernetes namespace in which Grafana is installed, _grafana-namespace_ if this differs from the namespace in which SAS Event Stream Processing is installed.
-   > **Caution**: Running the installation script might overwrite any existing Grafana configuration.
+   > **Caution**: Running the installation script will overwrite any existing Grafana configuration.
```
Let's change "will overwrite" to "overwrites". Advice is to use present tense whenever possible.

mtlljm committed:
(ESPSTUDIO-8275)
Plugin deployment option for different namespaces on Viya
make the readme more VIYA focused

Change-Id: Ib97c0df8bf9e133ae0eb3914fca3c7abd1707f77
